examples/datafusion.rs

```diff
@@ -64,7 +64,7 @@ impl SimpleQueryHandler for DfSessionService {
             Ok(vec![Response::Error(Box::new(ErrorInfo::new(
                 "ERROR".to_owned(),
                 "XX000".to_owned(),
-                "Datafusion is a readonly execution engine. To load data, call\nLOAD csv_file_path table_name;".to_owned(),
+                "Datafusion is a readonly execution engine. To load data, call\nLOAD json_file_path table_name;".to_owned(),
```

Comment:
By the way, could you please include a JSON sample file as reference?

Reply:
like this
```json
{"a":[1,2,3],"b":[2.0,1.3,-6.1],"c":[false, true]}
{"a":[4,5,7],"b":[1.6,6.9,2.34],"c":[true, true]}
{"a":[2,5,2],"b":[0.1,1.9,-2.3],"c":[false, false]}
````